Firestore/README.md

```diff
@@ -10,6 +12,42 @@
   * Select the Firestore_Tests_iOS scheme
   * ⌘-u to build and run the unit tests
 
+### Command-line builds
+
+You can also build from the command-line, though this requires a slightly
+different setup:
+
+```
+PLATFORM=iOS pod update --project-directory=Firestore/Example
+scripts/build.sh Firestore iOS
+```
+
+Note:
+  * `PLATFORM` here is specifying an environment variable that's active for the
+    `pod update` invocation.
+  * You can also use `macOS` or `tvOS` in place of `iOS` above.
+  * This will modify the Xcode project files; you'll need to revert these
+    changes to create a PR.
+
+The [issue](https://github.com/CocoaPods/CocoaPods/issues/8729) that requires
+this workaround is that Firestore's `Podfile` contains multiple platforms, and
+ever since Xcode 10.2, CocoaPods generates Xcode projects that are break by
+default when built by the `xcodebuild` command-line tool. There's a workaround
+possible that involves disabling Xcode's default mechanism of finding implicit
+dependencies, but this is something we'd have to disable Firebase-wide and
+there hasn't been an appetite to do this.
+
+### Swift package manager
+
+Firestore also supports building with Swift Package Manager. To build this way
+use:
```

Comment:
Or `open Package.swift` and choose the FirebaseFirestore scheme to build in Xcode.

Reply:
Having never done that, I hesitate to document it. Feel free to push a commit on top that adds that.

Reply:
OK, I'll either do that or wait until we're ready to address the Firestore SPM technical debt more broadly.